ci-operator/step-registry/redhat-appstudio/openshift-upgrade/redhat-appstudio-openshift-upgrade-workflow.yaml

```diff
@@ -0,0 +1,35 @@
+workflow:
+  as: redhat-appstudio-openshift-upgrade
+  steps:
+    # https://docs.ci.openshift.org/docs/architecture/step-registry/#marking-post-steps-best-effort
+    allow_best_effort_post_steps: true
+    # https://docs.ci.openshift.org/docs/architecture/step-registry/#skipping-post-steps-on-success
+    allow_skip_on_success: true
+    env:
+      # for "openshift-configure-cincinnati" step
+      CHANGE_CHANNEL_BASE: candidate
+      # for "redhat-appstudio-e2e" step
+      SKIP_BOOTSTRAP: "true"
+      E2E_TEST_SUITE_LABEL: "rhtap-demo"
```

Comment:
Thanks for the upgrade test automation. Running `rhtap-demo` suite after the upgrade complete will be testing there is no issue creating user workloads after the upgrade. Will it be better to also tests there is no impact for the existing users ? I mean creating user workloads before upgrade and see if those user workloads are not impacted by the upgrade.

Reply:
good idea, I created a [follow-up ticket for this](https://issues.redhat.com/browse/KONFLUX-2269)